### PR TITLE
PULL_REQUEST_TEMPLATE: add a note about usage of infinitive instead of past sense

### PR DESCRIPTION
**Pull request by sa2ajj**
## Contributor Checklist:

* [x] I have updated the unit tests -- *no need*
* [x] I have created a file in the `master/buildbot/newsfragments` directory (and read the `README.txt` in that directory) -- *no need*
* [x] I have updated the appropriate documentation -- *no need*

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,9 +9,14 @@ http://docs.buildbot.net/latest/developer/quickstart.html
 And especially:
 http://docs.buildbot.net/latest/developer/pull-request.html
 
-
 ## Contributor Checklist:
 
+*Important*: please use infinitive when describing changes- not "Added" but "Add": you are submitting a request to change, you have not change anything yet (in your branch, yes, in the target branch, no).
```
Comment (p12tic) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think it's best to move this to a separate section above that is to be removed. Usually most PRs leave the checkboxes in the PR description.

Comment (sa2ajj) [reply .github/PULL_REQUEST_TEMPLATE.md]: would something like "## Commit Message Guidelines" make sense?

Comment (tardyp) [reply .github/PULL_REQUEST_TEMPLATE.md]: We already have a documentation for pull-request, with lot of important guidelines in it.

http://docs.buildbot.net/latest/developer/pull-request.html

I'm not saying that this particular rule is less important than the others, but it should imho be managed as part of the others.

Comment (sa2ajj) [reply .github/PULL_REQUEST_TEMPLATE.md]: i overlooked that one :(  there's no point of repeating the same rules in different places...

**Pull request closed without merging**